### PR TITLE
Feat: implemented header , hero sections

### PR DESCRIPTION
**Pull request by hatim-developer**

### CODE REVIEW

File: src/containers/SkillList.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const SkillList = ({ skills }: { skills: string[] }) => {
+  return (
+    <ul className="md: mx-auto flex flex-row flex-wrap justify-center md:justify-start">
+      {skills.map((skill) => {
+        return (
+          <li className="bg-highlight-color/10 text-highlight-color mr-1.5 mt-2 rounded-full px-3 py-1 text-sm">
```
Comment (afkcodes) [inline src/containers/SkillList.tsx]: key ?

File: src/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import logo from "assets/img/logo.png";
+
+const Header = () => {
+  return (
+    <header className="sticky top-0">
+      <section className="mx-auto flex max-w-screen-lg flex-row flex-wrap items-center justify-between px-4 py-3">
```
Comment (afkcodes) [inline src/components/Header.tsx:6]: max-w-screen no need for this until your content is overflowing

File: src/components/Hero.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+          having 7+ years of experience in software industry developing the{" "}
+          <span className="text-secondary-color font-semibold">
+            beautiful websites, games and apps,{" "}
+          </span>
+          I find joy in creating
+          <span className="text-secondary-color font-semibold">
+            {" "}
+            web products{" "}
+          </span>
+          see there impact in society.
```
Comment (afkcodes) [inline src/components/Hero.tsx]: their

File: src/components/Header.tsx (hunk 2 above)
Comment (hatim-developer) [reply src/components/Header.tsx:6]: don't want the contents to go out beyond this max-w-screen-lg boundary, just setting the limit for vary wider screens

**Pull request merged**